driver/csiplugin/connectors/rest_v2.go

```diff
@@ -718,7 +721,33 @@ func (s *SpectrumRestV2) ListCSIIndependentFilesets(ctx context.Context, filesys
 		return nil, err
 	}
 
-	return getFilesetResponse.Filesets, nil
+	filesets := getFilesetResponse.Filesets
+
+	emptyPages := Pages{}
+	for getFilesetResponse.Paging != emptyPages {
+		lastID := getLastID(getFilesetResponse.Paging.Next)
```

Comment:
I feel we can get the lastId using getFilesetResponse.Paging.LastID directly ?

  "paging" : {
    "baseUrl" : "/scalemgmt/v2/filesystems/local-sample/filesets",
    "fields" : "filesetName",
    "lastId" : 1245,
    "next" : "/scalemgmt/v2/filesystems/local-sample/filesets?lastId=1245&fields=filesetName"
  },

Reply:
The struct Pages in CSI had only next field earlier so CSI was not reading other fields, have added other fields now and improved this part